lib/manageiq/schema/miq_command_recorder.rb

```diff
@@ -0,0 +1,21 @@
+module ManageIQ
+  module Schema
+    module MiqCommandRecorder
```

Comment:
super minor, but I think we can remove the Miq prefix from all of these since they are namespaced under the ManageIQ namespace anyway.

Reply:
100% agree with that.  Good point.  👍